ui/components/DataTable.tsx

```diff
@@ -134,14 +134,12 @@ export function filterRows<T>(rows: T[], filters: FilterConfig) {
 
     _.each(filters, (vals, category) => {
       let value;
-      // status
-      if (category === "status") {
-        value = filterByStatusCallback(row);
-      }
+
+      if (vals.transformFunc) value = vals.transformFunc(row);
       // strings
       else value = row[category];
 
-      if (!_.includes(vals, value)) {
+      if (!_.includes(vals.options, value)) {
         ok = false;
```

Comment:
❓ Do we have to go through remaining elements if this condition is true `!_.includes(vals.options, value)` and ok becomes false? Can we call `return false;` after `ok = false` to exit the Lodash `each` loop or if it does not work can we use a regular JS loop with break instead?

Reply:
that return makes sense - added it and nothing broke